### ISSUE TITLE
Video password verification failed on Vimeo protected video.

### ISSUE THREAD

**Issue by benjaminmikiten** [rg3/youtube-dl#5001]
Ran into this issue—know the password's correct, also doesn't work on single videos.

````
benjaminmikiten@benjaminmikiten ~/Dropbox/printing_vids $ youtube-dl --batch-file batch.txt --video-password dcp2015 -v
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--batch-file', 'batch.txt', '--video-password', u'PRIVATE', '-v']
[debug] Batch file urls: [u'https://vimeo.com/118076403', u'https://vimeo.com/118062456', u'https://vimeo.com/118072708', u'https://vimeo.com/118079046', u'https://vimeo.com/118080059', u'https://vimeo.com/118081777', u'https://vimeo.com/118085658', u'https://vimeo.com/118073801', u'https://vimeo.com/118086390', u'https://vimeo.com/118092186', u'https://vimeo.com/118092776']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.19.3
[debug] Python version 2.7.6 - Darwin-14.1.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg 2.5.4, ffprobe 2.5.4
[debug] Proxy map: {}
[vimeo] 118076403: Downloading webpage
[vimeo] 118076403: Extracting information
[vimeo] 118076403: Verifying the password
[vimeo] 118076403: Downloading webpage
[vimeo] 118076403: Extracting information
ERROR: video password verification failed!; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 264, in _real_extract
    flags=re.DOTALL)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 531, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract info section; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 269, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 275, in _real_extract
    smuggle_url(url, {'_video_password_verified': 'verified'}))
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 272, in _real_extract
    raise ExtractorError('video password verification failed!')
ExtractorError: video password verification failed!; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

````

**Comment by tedski** [comment]
Seeing the same on 2015.02.20:

```
$ youtube-dl --verbose --video-password "pa$$w0rd" https://vimeo.com/123456789
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', '--video-password', u'PRIVATE', 'https://vimeo.com/123456789']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.20
[debug] Python version 2.7.9 - Darwin-14.1.0-x86_64-i386-64bit
[debug] exe versions: none
[debug] Proxy map: {}
[vimeo] 119792545: Downloading webpage
[vimeo] 119792545: Extracting information
[vimeo] 119792545: Verifying the password
[vimeo] 119792545: Downloading webpage
[vimeo] 119792545: Extracting information
ERROR: video password verification failed!; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 264, in _real_extract
    flags=re.DOTALL)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 531, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract info section; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 269, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 275, in _real_extract
    smuggle_url(url, {'_video_password_verified': 'verified'}))
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 272, in _real_extract
    raise ExtractorError('video password verification failed!')
ExtractorError: video password verification failed!; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by dstftw** [comment]
Thanks for the report. Password protected videos will work in the next version.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.02.21. See [our FAQ](https://yt-dl.org/update) if you have trouble updating.

**Comment by tedski** [comment]
Confirmed this is no longer reproduced in 2015.02.21.  Thanks for the fix!
